tests/unit/test_utils.py

```diff
@@ -27,7 +27,7 @@
             }
         },
         {
-            "dataclasses-json": "dataclasses-json@ git+https://github.com/lidatong/dataclasses-json.git@v0.5.7"
+            "dataclasses-json": "dataclasses-json @ git+https://github.com/lidatong/dataclasses-json.git@v0.5.7"
```

Comment:
To match the exact spec of the pip docs but ended up being also inconsequential to fixing anything.